File: app/src/main/java/xyz/realms/mgit/ui/common/BindingHelper.java
package xyz.realms.mgit.ui.common;

import androidx.databinding.BindingAdapter;

import com.google.android.material.textfield.TextInputLayout;


public class BindingHelper {
    @BindingAdapter("errorText")
    public static void setErrorMessage(TextInputLayout view, String errorMessage) {
        if (errorMessage != null) {
            view.setError(errorMessage);
        }
    }
}

File: app/src/main/java/com/xinglan/mgit/common/BindingHelper.kt
package com.xinglan.mgit.common

import androidx.databinding.BindingAdapter
import com.google.android.material.textfield.TextInputLayout


@BindingAdapter("app:errorText")
fun setErrorMessage(view: TextInputLayout, errorMessage: String?) {
    errorMessage.let { view.error = errorMessage }
}

